recipe/build.sh

```diff
@@ -3,8 +3,8 @@
 CHOST=${macos_machine}
 
 FINAL_CPPFLAGS="-D_FORTIFY_SOURCE=2"
-FINAL_CFLAGS="-ftree-vectorize -fPIC -fstack-protector-strong -O2 -pipe"
-FINAL_CXXFLAGS="-ftree-vectorize -fPIC -fstack-protector-strong -O2 -pipe -stdlib=libc++ -fvisibility-inlines-hidden -fmessage-length=0"
+FINAL_CFLAGS="-fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -ftree-vectorize -fPIC -fstack-protector-strong -O2 -pipe"
+FINAL_CXXFLAGS="-fno-omit-frame-pointer -mno-omit-leaf-frame-pointer -ftree-vectorize -fPIC -fstack-protector-strong -O2 -pipe -stdlib=libc++ -fvisibility-inlines-hidden -fmessage-length=0"
```

Comment:
I'll have to read up on this, but at first glance, we don't want this in non CONDA_BUILD usage. We want to reduce the usage.

Reply:
If it shouldn't be in non-CONDA_BUILD usage, we would add it directly to https://github.com/conda-forge/clang-compiler-activation-feedstock/blob/5ce57c1c99e4a7eebe2cf67d814f723015b3ba2c/recipe/deactivate-clang.sh#L88 or should we introduce another variable?

Reply:
https://www.brendangregg.com/blog/2024-03-17/the-return-of-the-frame-pointers.html is also a good resource for reading up. It probably doesn't add much to the Fedora page, it's just a different perspective.